hopsworks-common/src/main/java/io/hops/hopsworks/common/project/ProjectController.java

```diff
@@ -2596,7 +2596,8 @@ public CertsDTO downloadCert(Integer projectId, Users user) throws ProjectExcept
       return new CertsDTO("jks", accessCredentials.getkStore(), accessCredentials.gettStore());
     } catch (Exception ex) {
       LOGGER.log(Level.SEVERE, null, ex);
-      throw new DatasetException(RESTCodes.DatasetErrorCode.DOWNLOAD_ERROR, Level.SEVERE, "projectId: " + projectId,
+      throw new DatasetException(RESTCodes.DatasetErrorCode.DOWNLOAD_ERROR, Level.SEVERE, "Failed to " +
+        "download project-user certificates for projectId: " + projectId + ", user: " + user.getUsername(),
```

Comment:
@kai-chi let's put the project name in the message as well

Reply:
@tkakantousis done